recipe/meta.yaml

```diff
@@ -18,19 +18,21 @@ build:
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
   host:
     - python
     - pip
     - cffi >=1.5.0
     - setuptools
     - wheel
-    - c-ares 1.17
+    - c-ares {{ c_ares }}
   run:
     - python
+    - c-ares # bounds through run_exports
     - cffi >=1.5.0
+  run_constrained:
```

Comment:
```suggestion
```
Since with provided pycares with idna in past versions, we should keep it for this reason.